File: C & C++/Problem_1.cpp
#include <iostream>
#include <string.h>
using namespace std;

int func(int num)
{
    int sum = 0;
    num -= 1;
    while (num > 0)
    {
        if (num % 3 == 0 || num % 5 == 0)
        {
            sum += num;
        }
        num -= 1;
    }
    return sum;
}

int main(int argc, char const *argv[])
{
    int num;
    cout << "Enter: ";
    cin >> num;
    cout << "Ans: " << func(num);
    return 0;
}

File: Python/Problem_1.py
def is_multiple(num):
    if num % 3 == 0 or num % 5 == 0:
        return True
    return False

def sum(num):
    x = 1
    s = 0
    while x < num:
        if is_multiple(x) is True:
            s += x
        x += 1
    del x
    return s

print(sum(1000))
